db/data.py

```diff
@@ -9,62 +9,67 @@ def get_id_from_scribble(data):
     data_json = json.dumps(data, sort_keys=True, separators=(',', ':'))
     data_sha256 = sha256(data_json.encode("utf-8")).hexdigest()
 
-    with db.new_connection() as conn:
-        with conn.cursor() as cursor:
-            cursor.execute("""SELECT s.gid
-                              FROM scribble s
-                         LEFT JOIN scribble_json sj
-                                ON sj.id = s.data
-                            WHERE sj.data_sha256 = %s""", (data_sha256, ))
-
-            if cursor.rowcount:
-                return cursor.fetchone()["gid"]
-            else:
-                return None
+    query = """SELECT s.gid
+                 FROM scribble s
```

Comment:
I think we can just proceed as is. I doubt we're going to be sued over a table name. I'm ok to merge this as is.